Rohit_assignment_1.ipynb

```diff
@@ -0,0 +1,1066 @@
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 45
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ANYwg3TgXQOQ",
+        "colab_type": "text"
+      },
+      "source": [
+        "#### Generate a *random* integer between 1 and 50\n",
+        "\n",
+        "`32`"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "_5zQOBImXQOT",
+        "colab_type": "code",
+        "colab": {},
+        "outputId": "aaf7a36f-a343-4f92-ede7-30f73735f6b9"
+      },
+      "source": [
+        "a10 = np.random.randint(1, 50)\n",
+        "a10"
+      ],
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "36"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 47
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "WDhy2BFOXQOb",
+        "colab_type": "text"
+      },
+      "source": [
+        "#### Create a Numpy 1d array consisting of five *random* integers between 1 and 50\n",
+        "\n",
+        "`array([36, 30, 32, 14,  2])`"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "tsljNldUXQOc",
+        "colab_type": "code",
+        "colab": {},
+        "outputId": "14723c71-0b7d-4ab2-faf6-7fb291529576"
+      },
+      "source": [
+        "a11 = np.random.randint(1, 50, size=(5))\n",
+        "a11"
+      ],
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "array([41, 43, 23, 28, 14])"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 48
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "tZcYx6kVXQOg",
+        "colab_type": "text"
+      },
+      "source": [
+        "#### Create a Numpy array consisting of 20 linearly spaced points between 0 and 1\n",
+        "\n",
+        "`array([ 0.        ,  0.05263158,  0.10526316,  0.15789474,  0.21052632,\n",
+        "        0.26315789,  0.31578947,  0.36842105,  0.42105263,  0.47368421,\n",
+        "        0.52631579,  0.57894737,  0.63157895,  0.68421053,  0.73684211,\n",
+        "        0.78947368,  0.84210526,  0.89473684,  0.94736842,  1.        ])`"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "41ed4-0rXQOh",
+        "colab_type": "code",
+        "colab": {},
+        "outputId": "7b3ad7ed-b2f0-4fbc-dd48-3af036bde2f8"
+      },
+      "source": [
+        "a12 = np.random.random(20)\n",
```

Comment:
### Create a Numpy array consisting of 20 linearly spaced points between 0 and 1

It is not asking about random numbers between 0 and 1.

Reply:
Thank you for pointing out the mistake. I have solved it now.